.github/workflows/check-python.yaml

```diff
@@ -100,19 +102,28 @@ jobs:
           cache: "poetry"
 
       - name: Install dependencies
-        run: poetry install --no-interaction
+        run: poetry install --no-interaction --with cicd
```

Comment:
I think we can remove --with cicd and Start LocalNet now

Reply:
Nice, done